src/Ethereum/ABI/ValueDecoder.h

```diff
@@ -19,7 +19,5 @@ class ValueDecoder {
     static uint256_t decodeUInt256(const Data& data);
     // Decode an arbitrary type, return value as string
     static std::string decodeValue(const Data& data, const std::string& type);
-    // Decode an array of given simple types; return each element as a string in a vector
-    static std::vector<std::string> decodeArray(const Data& data, const std::string& elementType);
```

Comment:
what hapenned here?

Reply:
`decodeValue` and `decodeArray` have almost the the same function bodies: [link](https://github.com/trustwallet/wallet-core/blob/5d0063ced7d07a1bbebf76b20ac25c16051afb53/src/Ethereum/ABI/ValueDecoder.cpp#L20-L43).
The only difference in the return type. But `decodeArray` is not used anywhere in C++ code, and it not exposed as an FFI, so I removed it